spec/unit/provider/package/yum_spec.rb

```diff
@@ -7,6 +7,13 @@
   include PuppetSpec::Fixtures
   it_behaves_like 'RHEL package provider', provider_class, 'yum'
 
+  it "should be less specific" do
```

Comment:
The spec name makes me wonder "less specific than what?".

Reply:
Perhaps I should say "should have lower specificity" like [this](https://github.com/puppetlabs/puppet/blob/5f4e2804521b39757057520553cf7871f3010191/spec/unit/type_spec.rb#L354)?  Otherwise, I would want to add a comment block with a paragraph about what that means, since I'm not sure how to make a short title.

Reply:
I'm cool with "lower specificity", since it's the term used in the code elsewhere